.gitpod.yml

```diff
@@ -0,0 +1,5 @@
+tasks:
+  - init: |
+      python -m pip install --upgrade pip
+      python -m pip install "tox<4.0" poetry
+      poetry install
```

Comment:
Do we also want to include the python test explorer plugin? https://www.gitpod.io/docs/introduction/languages/python#python-test-explorer

Reply:
Will do!